features/issue/components/EditIssueForm.tsx

```diff
@@ -74,12 +92,16 @@ export const EditIssueForm = () => {
 		fetchIssue().catch(error =>
 			setError(error.toString() || 'Something went wrong')
 		)
+		fetchUsers().catch(error =>
+			setError(error.toString() || 'Something went wrong')
+		)
 	}, [issueId, intIssueId])
 
 	const onSubmit = async (data: EditIssueFormData) => {
+		const submittedUsers = data.users?.map(user => ({ userId: user.value }))
 		startTransition(async () => {
 			setError('')
-			editIssue(intIssueId, data)
+			editIssue(intIssueId, { ...data, users: submittedUsers || [] })
```

Comment:
?? lepsze niż || we większości przypadków
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or
Tutaj masz czym to się różni